apps/sms/views/conversation/js/conversation.js

```diff
@@ -375,6 +374,32 @@ var ConversationView = {
     this.toggleRecipientSuggestions();
   },
 
+  setupComposeLock() {
+    var hasValidRecipients = false;
+    var hasEmailRecipients = false;
+
+    var validNumbers = this.recipients.numbers;
+
+    if (validNumbers.length) {
+      hasValidRecipients = true;
+      hasEmailRecipients = Settings.supportEmailRecipient &&
+        validNumbers.some(Utils.isEmailAddress);
+    } else {
+      var notValidatedNumber = this.recipients.inputValue;
+
+      hasEmailRecipients = Settings.supportEmailRecipient &&
```

Comment:
This line changes behavior a bit - earlier when you have some text in message and just entered "valid" number, but not yet confirmed - send button was enabled, but the same didn't work for email - no once email is recognized, send button would be enabled and message will be converted to mms. I'd say we should be consistent, but I don't remember it was done intentionally.